JDK8 is more strict than JDK7 when generating javadocs and this causes some
problems here:
https://www.redhat.com/archives/sclorg/2016-January/msg00010.html

Thanks for hint to Michal Srb.

diff -up org.eclipse.tycho-tycho-0.23.0/pom.xml.test1 org.eclipse.tycho-tycho-0.23.0/pom.xml
--- org.eclipse.tycho-tycho-0.23.0/pom.xml.test1	2015-05-28 17:14:06.000000000 +0200
+++ org.eclipse.tycho-tycho-0.23.0/pom.xml	2016-01-06 21:56:59.636213557 +0100
@@ -73,6 +72,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<min.jdk.version>1.7</min.jdk.version>
 
+		<additionalparam>-Xdoclint:none</additionalparam>
 		<!-- copy&paste from maven/pom.xml file. I wish import scope worked in 3.0 -->
 		<classWorldsVersion>2.2.3</classWorldsVersion>
 		<commonsCliVersion>1.2</commonsCliVersion>
